e2e-tests/README.md

```diff
@@ -60,6 +60,29 @@ Then you can run the tests against staging using the following command
 
 If you wish to run them against different environemts, set `PYTEST_BASE_URL` as indicated in the sections above for running tests against localhost or production
 
+___Running tests using headless Firefox___
+
+NOTE: This instructions are for Linux and macOS only.
+
+To run the tests using a copy of Firefox that can be run in 'headless' mode (meaning with no UI), do the following:
+
+Set an environment variable MOZ_HEADLESS to be '1'. Check the documentation for your shell on how to do this. For zsh you can do using the following command:
+
+	$ export MOZ_HEADLESS=1
+
+Next you need to modify `tests/conftest.py` and add the following pytest fixture to tell it to use a different version of Firefox. Check to verify the version you want to use supports running headless.
+
+	@pytest.fixture
+	def firefox_options(firefox_options):
+	    firefox_options.binary = '/path/to/firefox/binary'
+	    return firefox_options
```

Comment:
Pretty sure you can do this on the command line with `--firefox-path` or something like that. Is that not true anymore?

Reply:
Oh, I see. It's an argument for pytest--not tox. Nevermind.